### PR TITLE
add heapster addon

### PR DESCRIPTION
**Pull request by sethpollack**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kops/2483)

### CODE REVIEW

**Comment by archSeer** [comment]
`addon.yaml` needs to be updated: https://github.com/sethpollack/kops/blob/881ca9e01f351047e199dc1492e8f2da26a981c1/addons/monitoring-standalone/addon.yaml

**Comment by sethpollack** [comment]
@archSeer I was following the lead from https://github.com/kubernetes/kops/pull/2480

**Comment by chrislovecnm** [comment]
@k8s-bot ok to test

File: addons/monitoring-standalone/v1.6.0.yaml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+          command:
+            - /pod_nanny
+            - --cpu=80m
+            - --extra-cpu=0.5m
+            - --memory=140Mi
+            - --extra-memory=4Mi
+            - --threshold=5
+            - --deployment=heapster
+            - --container=heapster
+            - --poll-period=300000
```
Comment (chrislovecnm) [inline addons/monitoring-standalone/v1.6.0.yaml]: This is standalone without data persistence?

**Comment by chrislovecnm** [comment]
We do not have a readme, for this folder, do you want to file and issue or do one?

File: addons/monitoring-standalone/v1.6.0.yaml (hunk 1 above)
Comment (chrislovecnm) [inline addons/monitoring-standalone/v1.6.0.yaml]: Never mind - figure it out

**Comment by chrislovecnm** [comment]
Waiting on e2e and had a question.

**Comment by chrislovecnm** [comment]
@justinsb what is the call on https://github.com/kubernetes/kops/pull/2483#issuecomment-298975259

**Comment by chrislovecnm** [comment]
@archSeer do you know how the addon.yaml is used in the code base?  It is a mystery to me :)  Have not dug through it.

**Comment by archSeer** [comment]
@chrislovecnm It's briefly described here: https://github.com/kubernetes/kops/blob/master/docs/addon_manager.md#versioning

I don't really understand the purpose of using kops addons instead of helm though (config management is manual etc.), especially since helm charts are maintained by a larger user group. Having to maintain these addons is a bit tedious, and they quickly fall behind as we can see here -- 3 major versions behind. See #2391, would be good to get some form of an official statement on kops addons vs helm).

**Comment by chrislovecnm** [comment]
waiting on CI

**Comment by justinsb** [comment]
@archSeer you're definitely free to use helm for these addons.  We'll hopefully be unifying everything, as a project, soon :-)

**Pull request merged**